pkg/util/log/crash_reporting.go

```diff
@@ -87,9 +88,16 @@ func ReportPanic(ctx context.Context, r interface{}, depth int) {
 	// at least somewhat helpful in telling us where crashes are coming from.
 	// We capture the full stacktrace below, so we only need the short file and
 	// line here help uniquely identify the error.
-	file, line, _ := caller.Lookup(depth + 3)
-
-	reportable := fmt.Sprintf("%s %s:%d", format(r), filepath.Base(file), line)
+	// Some exceptions, like a runtime.Error, are assumed to be fine as-is.
```

Comment:
What is the basis for this assumption? None of the runtime errors I can find currently include any potential user data, but I don't see any commitment that this will always be the case.